html/help.html

```diff
@@ -0,0 +1,30 @@
+<html>
+		<head>
```

Comment:
(and, indentation in html is not a big deal, just want to be able to read it on github nicely)